<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellForm</class>
 <widget class="QWidget" name="CellForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="comboBox_q"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_a"/>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_d"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
